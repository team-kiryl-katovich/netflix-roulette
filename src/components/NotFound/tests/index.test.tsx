import * as React from 'react';
import * as enzyme from 'enzyme';

import { NotFound } from '..';

describe('[COMPONENTS] [NOT_FOUND]', () => {
  test('should render component', () => {
    const component = enzyme.shallow(<NotFound />);
    expect(component).toMatchSnapshot();
  });
});
