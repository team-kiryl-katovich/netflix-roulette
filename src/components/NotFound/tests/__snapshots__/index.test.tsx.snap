// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENTS] [NOT_FOUND] should render component 1`] = `
<div
  className="page-not-found"
>
  <Component
    className="page-not-found__label"
    text="Page not found"
  />
</div>
`;
